### DISCUSSION TITLE
Problems setting reverse domain

### DISCUSSION THREAD

**Discussion by stachovski** [coder/code-server#3634] [General]
Hi everyone, 
I'm trying to setup code server to use a subpath instead my domain itself

I tried using the example on the guide with no success.

`stachovski.de/code/* {
  uri strip_prefix /code
  reverse_proxy 127.0.0.1:8080
}
`

Can anyone help me?

**Comment by jsjoeio** [comment]
Are you using NGINX or something? Have you read [the docs](https://github.com/cdr/code-server/blob/main/docs/guide.md#nginx)?

Or a better question, what does your setup look like?
